apps/passport-client/components/screens/ProveScreen/GenericProveSection.tsx

```diff
@@ -42,6 +43,7 @@ export function GenericProveSection<T extends PCDPackage = PCDPackage>({
 
   const onProveClick = useCallback(async () => {
     setProving(true);
+    setError(undefined);
```

Comment:
maybe we should `setError(undefined)` whenever any of the form fields change, not just when we click prove

Reply:
done